game.js

```diff
@@ -0,0 +1,53 @@
+
+var name = 'tom';
+var city = 'boulder';
+var team = 'seahawks';
+var correctAnswers = 0;
+// var counter = 0;
```

Comment:
It's good practice to remove unused, commented code before you deploy. No points off or anything, just a tip!